src/wow.coffee

```diff
@@ -15,6 +15,26 @@ class Util
   isMobile: (agent) ->
     /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(agent)
 
+  createEvent: (event, bubble = false, cancel = false, detail = null) ->
+    if document.createEvent? # W3C DOM
+      customEvent = document.createEvent('CustomEvent')
+      customEvent.initCustomEvent(event, bubble, cancel, detail)
+    else if document.createEventObject? # IE DOM < 9
+      customEvent = document.createEventObject()
+      customEvent.eventType = event
+    else
+      customEvent.eventName = event
+
+    customEvent
+
+  emitEvent: (elem, event) ->
+    if elem.dispatchEvent? # W3C DOM
+      elem.dispatchEvent(event)
+    else if elem[event]? 
+      elem[event]();
+    else if elem['on'+event]?
```

Comment:
Same here, I think `else if "on{event}" of elem` would work. Also, use string interpolation instead of string concatenation when possible (merely for consistency).